integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -336,22 +335,19 @@ private int readdirInternal(String path, long buff, long filter, long offset,
   @Override
   public int open(String path, FuseFileInfo fi) {
     final int flags = fi.flags.get();
-    // TODO(maobaolong): Add an option to decide whether reject rw flag
-    //  or fallback to truncate
-    // 0b11 here aimed to handle both 0x8001 and 0x8002
-    // 0x8001 stand for 'w' which means open file for writing.
-    // 0x8002 stand for 'r+' which means Open file for reading and writing.
-    boolean overwrite = OpenFlags.valueOf(flags) == OpenFlags.O_WRONLY
-        || (flags & 0b11) != 0;
-    String methodName = overwrite ? "Fuse.OpenOverwrite" : "Fuse.Open";
-    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi, overwrite),
-        methodName, "path=%s,flags=0x%x", path, flags);
-  }
-
-  private int openInternal(String path, FuseFileInfo fi, boolean overwrite) {
+    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi),
+        "Fuse.open", "path=%s,flags=0x%x", path, flags);
+  }
+
+  private int openInternal(String path, FuseFileInfo fi) {
+    final int flags = fi.flags.get();
+    boolean overwrite = AlluxioFuseUtils.isOpenOverwrite(flags);
     final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
     try {
       if (overwrite) {
+        LOG.warn(String.format("Open path %s with flags 0x%x for overwriting. "
```

Comment:
oh you are talking about a lot of small files, yeah, if it's a log of small files, it may fill the log. but i do want users to know what this is not a common case and be aware. Any thoughts how to log those?

Reply:
it will be better to show all the fuse warning in a special place, like what we did with `bin/alluxio fsadmin docter`